Doc/library/time.rst

```diff
@@ -230,6 +231,10 @@ Functions
    returned by :func:`.time` is used.  ``ctime(secs)`` is equivalent to
    ``asctime(localtime(secs))``. Locale information is not used by :func:`ctime`.
 
+   .. note::
```

Comment:
I would prefer to mimick asctime() doc:

Convert a time (...) to a string representing local time of the following form: ``'Sun Jun  5 23:21:05 1993'``.